### PR TITLE
Add method getPrivateContainer

### PR DESCRIPTION
**Pull request by digitalsadhu**
Added as a separate method so as not to break
backwards compatibility with apps using getContainer

@pnw

### CODE REVIEW

**Comment by pnw** [comment]
@digitalsadhu Re: not breaking backwards compatibility. How else were you thinking of doing this? Add an `options` object to `getContainer` that accepts a `private` boolean?

**Comment by digitalsadhu** [comment]
Yea exactly. Could actually do that in a non breaking change way I suppose.

**Comment by pnw** [comment]
Maybe just make the breaking change and bump the major version? In any case, I wonder which way is actually "cleaner": having a single method with a parameter, or two separate methods. I haven't had a need for it, so I don't have any use cases.

**Comment by digitalsadhu** [comment]
@pnw How about the way I've done it in the latest commit then?

**Comment by pnw** [comment]
@digitalsadhu sorry, I never got back to you on this. Yeah I like that. There's probably no reason to publicly expose `getContainerWithMethod` is there?

File: trim.production.js [hunk 1]
```diff
@@ -71,23 +71,32 @@ function getDocument (trimId, callback) {
...
+function getContainer (trimId, callback) {
+  getContainerUsingMethod('GetContainer', trimId, token, callback);
+}
+
+/**
+ * @param trimId
+ * @param {containerCallback}  callback
+ */
+function getPrivateContainer (trimId, callback) {
+  getContainerUsingMethod('getPrivateContainer', trimId, token, callback);
```
Comment (markstuart) [inline trim.production.js]: The casing seems inconsistent between these (`Get` vs `get`). Does it not matter? Or do the two trim API methods actually differ in that way?

Comment (pnw) [inline trim.production.js]: Id assume those are the literal trim methods. I'll check the docs tom gave us. The api is hand crafted by someone at mdc and they're pretty inconsistent so I'd assume they aren't following any conventions

**Comment by digitalsadhu** [comment]
@pnw don't think so.

**Comment by digitalsadhu** [comment]
@pnw @anotheredward We all good to merge this? Its up to date with master so should just be a simple fast forward to merge

**Comment by anotheredward** [comment]
LGTM

**Comment by ebuckley** [comment]
LGTM

**Pull request merged**